src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/DefaultWorkspaceDirectoryPathResolverTest.cs

```diff
@@ -18,10 +18,12 @@ public void Resolve_RootUriUnavailable_UsesRootPath()
         {
             // Arrange
             var expectedWorkspaceDirectory = "/testpath";
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
Out of curiosity, why is this marked as obsolete? Is there an alternative we should be using instead?

Reply:
`RootPath` is Obsolete, but usage of it is specifically what we're testing so using an alternative is not an option.